Kernel/Devices/VMWareBackdoor.cpp

```diff
@@ -144,18 +141,30 @@ void VMWareBackdoor::disable_absolute_vmmouse()
 
 void VMWareBackdoor::send_highbandwidth(VMWareCommand& command)
 {
-    if (supported())
+    if (supported()) {
         IO::vmware_highbandwidth_send(command);
+#ifdef VMWAREBACKDOOR_DEBUG
+        dbgprintf("VMWareBackdoor Highbandwidth Send Results: EAX %x, EBX %x, ECX %x, EDX %x\n", command.ax, command.bx, command.cx, command.dx);
```

Comment:
Please use `dbg()` instead of `dbgprintf()` in new code.
I'd like to get to a point where we have no format strings at all in the kernel, but instead only type-safe `LogStream` code.